scripts/k8s_vm_template/FilesToPlace/containerd.certs.d/README.md

```diff
@@ -0,0 +1,21 @@
+# Containerd Registry Configuration
+
+As of containerd v1.5, registry configuration is handled with `config_path`. See [documentation](https://github.com/containerd/containerd/blob/main/docs/hosts.md#registry-configuration---introduction).
+
+For each registry, create a `<registry-name>/hosts.toml` file. For example if you have a mirror for Docker Hub, create `docker.io/hosts.toml` with:
+
+```toml
+server = "https://registry-1.docker.io"
+
+[host."https://my-mirror.example.com"]
+  capabilities = ["pull", "resolve"]
+  [host."https://my-mirror.example.com".header]
+    authorization = ["Basic <BASE64_ENCODED_CREDENTIALS>"]
```

Comment:
Could you add something to the .gitignore so nobody accidentally commits base64 credentials?